lib/widgets/fetchErrorHandler.dart

```diff
@@ -0,0 +1,124 @@
+// ignore: file_names
+import 'package:trufi_core/blocs/app_review_cubit.dart';
+import 'package:trufi_core/blocs/home_page_cubit.dart';
+import 'package:trufi_core/blocs/location_provider_cubit.dart';
+import 'package:trufi_core/blocs/preferences_cubit.dart';
+import 'package:trufi_core/l10n/trufi_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:trufi_core/repository/exception/fetch_online_exception.dart';
+import 'package:trufi_core/trufi_configuration.dart';
+import 'package:url_launcher/url_launcher.dart';
+// ignore: implementation_imports
+import 'package:provider/src/provider.dart';
+import 'package:package_info/package_info.dart';
+import 'package:latlong/latlong.dart';
+import 'alerts.dart';
+
+void onFetchError(BuildContext context, dynamic exception) {
```

Comment:
I think we could need some tests for this file.

Reply:
@md-weber  They are error pop-ups, I am sure how testable it is, could you help us on it?

Reply:
Sure I will have a look :)

Reply:
As discussed we would not test this part for now.